src/Cardano/CLI/Query.hs

```diff
@@ -195,31 +252,33 @@ queryVotingFunds
 queryVotingFunds nw mSlotNo (Api.Lovelace threshold) = do
   info <- queryVoteRegistrationInfo nw mSlotNo
 
-  let info' = filterAmounts (> threshold) info
+  funds <- fmap mconcat . forM (causeSumAmounts info) $ \(votepub, amt) -> do
+      if amt < fromIntegral threshold
```

Comment:
I would tend to use `partition` here - makes the next lines easier.

Reply:
Addressed in 6ebc251.